x-pack/plugins/timelines/server/search_strategy/index_fields/index.ts

```diff
@@ -91,6 +91,7 @@ export const requestIndexFieldSearch = async (
   const dataViewService = await indexPatterns.indexPatternsServiceFactory(
     savedObjectsClient,
     esClient.asCurrentUser
+    // todo does this need to pass the request?
```

Comment:
@stephmilovic do you know if we the request here?

Reply:
Seems like this is only doing a `get`.

Reply:
exactly we do not need it here